### DISCUSSION TITLE
Is it possible to rebroadcast camera RTSP streams instead of RTMP? How?

### DISCUSSION THREAD

**Discussion by dspears0809** [blakeblackshear/frigate#2396] [Ask A Question]
Is it possible to locally rebroadcast the RTSP incoming main RTSP stream locally on HA instead of RTMP?
It appears WebRTC only supports RTSP and not RTMP. I'd like send my six incoming RTSP streams to WebRTC for live Lovelace views at full framerate alongside Frigate.

**Comment by dspears0809** [comment]
Just saw this - not sure how I missed it earlier in the week...
https://github.com/dermotduffy/frigate-hass-card
I'll give this a try instead of the WebRTC addon :-)

EDIT: tried it, but seems it just wraps the existing WebRTC which still has the restriction of require an RTSP stream. :-(

So, back to the original question, is there a way to get RTSP streams out of Frigate (my HA box) instead of RTMP?

Thanks!

**Comment by Bod1985** [comment]
https://github.com/blakeblackshear/frigate/pull/1927

Part of the listed yet to be completed changes for 0.10 is changing from nginx RTMP to https://github.com/aler9/rtsp-simple-server

I’m not certain but I’d presume this change might give the functionality you’re after.

**Reply by dspears0809** [reply]
Hi, Bod1985,

Thanks for the quick response. Yes, it certainly will! As a temporary solution, I've setup Live555 proxy addon to copy four main/sub streams (QHD and HD) and simply refer to those in Frigate and WebRTC. Seems to be working fine so far with very little impact on NUC6 J3455 CPU, but I'd definitely prefer a feature inside Frigate to handle RTSP stream proxy with multiple output options! Thanks again for the quick insights! Look forward to upcoming release!
